generator/C/include_v2.0/mavlink_helpers.h

```diff
@@ -779,7 +779,9 @@ MAVLINK_HELPER uint8_t mavlink_frame_char_buffer(mavlink_message_t* rxmsg,
 				}
 			}
 			status->parse_state = MAVLINK_PARSE_STATE_IDLE;
-			memcpy(r_message, rxmsg, sizeof(mavlink_message_t));
+			if(NULL != r_message){
```

Comment:
Please don't reverse the order of comparisons.

Also, a space between the if and opening pren...

... and between the closing pron and opening brace.

Reply:
Implemented the suggested changes in commit 10b6e1fe96550e4cd7655f04c72618deedb47227

(It appears the date on my VM was wrong after resuming, so the commit shows up as from before this comment)

The order of comparisons was done to be consistent with the other null checks. In this commit, I updated the other null comparisons to not be reversed either.